src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const bookId = Date.now();
+
+export const Book = {
+  id: bookId,
+  title: 'APIs: A Strategy Guide',
+  author: 'Daniel Jacobson',
+};
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== bookId);
```

Comment:
- [ ] For the REMOVE_BOOK action the comparison should be made with the action.payload. The return statement should look like this `return state.filter(book => book.id !== action.payload);`